src/main/java/com/smartbear/readyapi/client/execution/CodegenBasedTestServerApi.java

```diff
@@ -282,4 +290,82 @@ private GenericType getReturnTypeHarLogRoot() {
     public void setBasePath(String basePath) {
         apiClient.setBasePath(basePath);
     }
+
+    @Override
+    public ProjectResultReport postProject(File file, boolean async, HttpBasicAuth auth) throws ApiException {
```

Comment:
We should add parameters to specify the test suite and test case names. We support that in the server API.